### DISCUSSION TITLE
文档：贡献指南中的这个表述是否有问题

### DISCUSSION THREAD

**Discussion by kcfuler** [umijs/umi#11596] [Q&A]
![image](https://github.com/umijs/umi/assets/87866012/da995582-cf5c-4158-881d-1064b82ffc27)

**Comment by fz6m** [comment]
欢迎 PR 修正。

**Reply by kcfuler** [reply]
https://github.com/umijs/umi/pull/11599#issuecomment-1704045182，改了一点，这样的提交符合规范吗

**Reply by kcfuler** [reply]
![image](https://github.com/umijs/umi/assets/87866012/6509443d-76b2-4c01-b971-fdd884b4fe99)
因为项目中的这个链接失效了
